src/app/zone-ingresses/sources.ts

```diff
@@ -38,8 +39,11 @@ export const sources = (api: KumaApi) => {
       res.items = res.items.filter((item) => {
         return item.zoneIngress.zone === name
       })
-      res.total = res.items.length
-      return res
+      return {
+        ...res,
+        total: res.items.length,
```

Comment:
**Suggestion**: Can we leave a note here saying that this needs to change once we can filter this API by Zone name? Currently, this is (both in `master` and in this PR) incorrect if there is more than one page of results (total != items.length).

Reply:
I added some comment here and in egresses.

Because we are filtering on the frontend. We kinda accepted that for the moment that is ok seeing as ingresses/egresses for a specific zone is never really gonna get past page 1.